objectRecognitionSource/src/main/java/io/vantiq/extsrc/objectRecognition/neuralNet/YoloProcessor.java

```diff
@@ -315,6 +389,45 @@ public NeuralNetResults processImage(byte[] image, Map<String, ?> request) throw
         results.setResults(foundObjects);
         return results;
     }
+    
+    /**
+     * A helper method used to crop images before they are run through the YOLO Processor, if specified in the
+     * source configuration or as query parameters. Returns the original image if an exception was caught while
+     * cropping.
+     * @param image     The byte array representation of the captured image
+     * @param x         The top left x coordinate
+     * @param y         The top left y coordinate
+     * @param w         The width of the "sub image"
+     * @param h         The height of the "sub image"
+     * @return          The cropped image resulting from the parameters above
+     */
+    public byte[] cropImage(byte[] image, int x, int y, int w, int h) {
```

Comment:
Cropping doesn't seem like something that's YOLO specific, is it?  Would this be better in some possibly abstract superclass that YOLO & any other processor could extend to get this capability?